pilot/cmd/pilot-agent/main.go

```diff
@@ -201,6 +207,13 @@ var (
 				tlsServerCertChain, tlsServerKey, tlsServerRootCert,
 				tlsClientCertChain, tlsClientKey, tlsClientRootCert,
 			}
+			dnsCertPath := env.RegisterStringVar(bootstrap.IstioMetaPrefix+model.NodeMetadataDNSCert, "", "").Get()
```

Comment:
What are the second and third parameters for?

Reply:
Default value and description
```func RegisterStringVar(name string, defaultValue string, description string) StringVar {```

https://github.com/istio/pkg/blob/5c2f5ef63692200f8504dfbcf7db5f67aaef806e/env/var.go#L111